main.js

```diff
@@ -1 +1,10 @@
-
+function toggleToDo(element){
+  var newClass;
+  var completed = element.querySelector(".fa").classList.contains("fa-check-square");
+  if (completed) {
+    newClass = element.querySelector(".fa").className.replace("fa-check-square", "fa-square-o");
```

Comment:
@ChrisBaidoo, Could you find a way to do this in a different way, to not use an `if` statement?

Reply:
@samdemaeyer Great news, the task has been completed. Another good news is that I understand the concept of DOM. Happy days!